src/Sulu/Bundle/PageBundle/Markup/Link/ExternalLinkProvider.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle\Markup\Link;
+
+use Ramsey\Uuid\Uuid;
+use Sulu\Bundle\MarkupBundle\Markup\Link\LinkConfigurationBuilder;
+use Sulu\Bundle\MarkupBundle\Markup\Link\LinkItem;
+use Sulu\Bundle\MarkupBundle\Markup\Link\LinkProviderInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class ExternalLinkProvider implements LinkProviderInterface
+{
+    public function __construct(
+        private TranslatorInterface $translator,
+    ) {
+    }
+
+    public function getConfiguration()
+    {
+        /** @var LinkConfigurationBuilder $linkConfigurationBuilder */
+        $linkConfigurationBuilder = LinkConfigurationBuilder::create();
+
+        return $linkConfigurationBuilder
+            ->setTitle($this->translator->trans('sulu_admin.external_link', [], 'admin'))
+            ->setResourceKey('')
+            ->setListAdapter('')
+            ->setDisplayProperties([])
+            ->setOverlayTitle('')
+            ->setEmptyText('')
+            ->setIcon('');
+    }
+
+    public function preload(array $hrefs, $locale, $published = true)
+    {
+        if (0 === \count($hrefs)) {
+            return [];
+        }
+
+        $result = [];
+        foreach ($hrefs as $href) {
+            $result[] = new LinkItem(Uuid::uuid4()->toString(), $href, $href, true);
```

Comment:
generate a uuid feels strange here, would just use the href as id.

Also not sure about the `title` that feels wrong from my point of view:


```suggestion
            $result[] = new LinkItem($href, '', $href, true);
```